src/stores/Binding/types.ts

```diff
@@ -57,10 +58,17 @@ export interface BindingChanges {
     addedCollections: string[];
 }
 
+export type BindingDisableUpdate = {
+    uuid: string;
+    bindingIndex: number;
+    val: boolean;
+};
```

Comment:
How does this type differ from `ToggleDisableUpdate`?

Reply:
It doesn't. I initially had two cause I wasn't sure if the shape would stay the same.